pysat/utils/_core.py

```diff
@@ -0,0 +1,410 @@
+from __future__ import print_function
+from __future__ import absolute_import
+
+import numpy as np
+
+
+def set_data_dir(path=None, store=None):
+    """
+    Set the top level directory pysat uses to look for data and reload.
+
+    Parameters
+    ----------
+    path : string
+        valid path to directory pysat uses to look for data
+    store : bool
+        if True, store data directory for future runs
+    """
+    import sys
+    import os
+    import pysat
+    if sys.version_info[0] >= 3:
+        if sys.version_info[1] < 4:
+            import imp
+            re_load = imp.reload
+        else:
+            import importlib
+            re_load = importlib.reload
+    else:
+        re_load = reload
+    if store is None:
+        store = True
+    if os.path.isdir(path):
+        if store:
+            with open(os.path.join(os.path.expanduser('~'), '.pysat',
+                                   'data_path.txt'), 'w') as f:
+                f.write(path)
+        pysat.data_dir = path
+        pysat._files = re_load(pysat._files)
+        pysat._instrument = re_load(pysat._instrument)
+    else:
+        raise ValueError('Path %s does not lead to a valid directory.' % path)
+
+
+def computational_form(data):
+    """
+    Input Series of numbers, Series, or DataFrames repackaged
```

Comment:
This needs a slightly better description.

Reply:
attempted a better description.  Difficult because I'm not really sure what this is for.   @rstoneback?